src/main/java/org/sagebionetworks/web/client/widget/ReactComponent.java

```diff
@@ -50,37 +69,32 @@ public void run() {
     t.schedule(0);
   }
 
+  public void render(ReactElement<?, ?> reactElement) {
+    this.reactElement = reactElement;
+    createRootAndRender();
+  }
```

Comment:
a bit of refactoring to unify the calls to `Timer.run`/`Timer.schedule` (`setTimeout`) and make it more clear why it's being scheduled. No change